js_modules/dagit/packages/core/src/graphql/types.ts

```diff
@@ -415,6 +415,7 @@ export type DagitMutation = {
   cancelPartitionBackfill: CancelBackfillResult;
   deletePipelineRun: DeletePipelineRunResult;
   deleteRun: DeletePipelineRunResult;
+  evaluateSensor: EvaluateSensorMutation;
```

Comment:
i think the naming here should be precise enough to leave no doubt as to what its doing. "evaluate" is vague enough to be interpreted either way. "dry run" is one idea [1]

Reply:
I actually really like dry run - I was having trouble encapsulating what's happening here - but I think that does it well